roles/science/tasks/mrtrix-dev.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - /opt/quarantine/software/mrtrix3/dev
+
+- name: Install mtrix3-dev dependencies
+  ansible.builtin.apt:
+    update_cache: true
+    state: present
+    autoclean: true
+    autoremove: true
+    pkg:
+      - python3
+      - libeigen3-dev
+      - zlib1g-dev
+      - libqt5opengl5-dev
+      - libqt5svg5-dev
+      - libgl1-mesa-dev
+      - libfftw3-dev
+      - libtiff5-dev
+      - libpng-dev
```

Comment:
Confirm up-to-date with:
https://mrtrix.readthedocs.io/en/latest/installation/build_from_source.html#linux